packages/core/dao/NemDAO.js

```diff
@@ -84,55 +72,16 @@ export default class NemDAO extends EventEmitter {
     return this._nemProvider.getAccountBalances(this._namespace)
   }
 
-  async getAccountBalance () {
-    const balance = await this.getAccountBalances()
-    return balance ? balance.unconfirmed : null
-  }
-
-  accept (transfer: TransferExecModel) {
-    setImmediate(() => {
-      this.emit('accept', transfer)
-    })
-  }
-
-  reject (transfer: TransferExecModel) {
-    setImmediate(() => {
-      this.emit('reject', transfer)
-    })
+  getAccountBalance () {
```

Comment:
Please add a comment, that getAccountBalance (wrapper for getAccountBalances) is required for uniformity os DAOs